src/Entity/HearingPostRequest.php

```diff
@@ -131,26 +131,26 @@ public function removeHearingRecipient(HearingRecipient $hearingRecipient): self
         return $this;
     }
 
-    public function getBriefExtraParties(): ?string
+    public function getBriefing(): ?HearingBriefing
     {
-        return $this->briefExtraParties;
+        return $this->briefing;
     }
 
-    public function setBriefExtraParties(string $briefExtraParties): self
+    public function setBriefing(?HearingBriefing $briefing): self
     {
-        $this->briefExtraParties = $briefExtraParties;
+        $this->briefing = $briefing;
 
         return $this;
     }
 
-    public function getBriefing(): ?HearingBriefing
+    public function getShouldSendBriefing(): bool
```

Comment:
Should we call this function `shouldSendBriefing` to make its use more linguistically pleasing, e.g.

```php
if ($request->shouldSendBriefing()) {
  …
}
```

?